composer.json

```diff
@@ -2,9 +2,9 @@
     "name": "keboola/billing-api-php-client",
     "license": "MIT",
     "require": {
-        "php": ">=5.6",
```

Comment:
Nikde se to na starym PHP nepouziva. A stejne by to tam ani s novym SAPI clientem neslo